cores/esp8266/Arduino.h

```diff
@@ -275,10 +275,13 @@ long secureRandom(long);
 long secureRandom(long, long);
 long map(long, long, long, long, long);
 
-extern "C" void configTime(long timezone, int daylightOffset_sec,
-    const char* server1, const char* server2 = nullptr, const char* server3 = nullptr);
+void configTime(int timezone, int daylightOffset_sec, const char* server1,
```

Comment:
So the prototype was only defined if _cpp was defined, but it was done with an `extern C` linkage.  What a mess, thanks for cleaning up.